app/api/v1/views/politicalparty.py

```diff
@@ -20,50 +20,64 @@ def create_political_party():
 
             # Return Party Data with status code 201: Created
             return make_response(jsonify({
-            'id': feedback["party_id"],
-            'name': feedback["name"]
-            }), 201)
+            "status" : 201,
+            'Message': 'Success! Party Created',
+            "Party": {
+                'id': feedback["party_id"],
+                'name': feedback["name"]
+            }}), 201)
         else:
-            return make_response(
-            jsonify({'error': 'Party already exists!'}),
-            416)
-
+            return make_response(jsonify({
+            "status" : 409,
+            'error': 'Party already exists!',
+            "Party": party }),
+            409)
 
     @politicalparty.route('/parties/<int:party_id>',methods=['PATCH'])
     def edit_political_party(party_id):
         party_data = request.get_json()
         party = Political().edit_political_party(party_data, party_id)
         return make_response(jsonify({
-            "message": "Success!! Party patched",
-            "data": party
-        }))
+            "status" : 200,
+            "message": "Success!! Party updated",
+            "Party": party
+        }),200)
 
 
     @politicalparty.route('/parties/<int:party_id>',methods=['DELETE'])
     def delete_political_party(party_id):
         party = Political().delete_political_party(party_id)
         if party:
             return jsonify({
+            "status" : 200,
             "message" : "Success!! Party Deleted",
             "party":party
-            })
-        return jsonify({
-            "message": "Error!! Not Deleted"
+            },200)
+        else:
+            return jsonify({
+            "message": "Error!! Party Not Deleted",
         })
 
     @politicalparty.route('/parties/<int:party_id>',methods=['GET'])
     def get_specific_political_party(party_id):
         party = Political().get_specific_political_party(party_id)
         return make_response(jsonify({
+            "status" : 200,
             "message": "Success!! Party found",
             "party":party
-        }))
+        }),200)
 
     @politicalparty.route('/parties',methods=['GET'])
     def get_political_parties():
-        parties =[]
         parties =Political().get_political_parties()
-        return make_response(jsonify({
-            "message": "Success!! Parties Listed",
-            "parties" : parties
-        }))
+        if len(parties) == 0:
+            return make_response(jsonify({
+                "status" : 404,
```

Comment:
Please return `200` for success because the request was successful only that it found an empty list. `404` is suitable when searching an item by id when such an id is not found.

Reply:
Updated